fastlane/lanes/release.rb

```diff
@@ -280,8 +280,10 @@
     apps = options[:app].nil? ? ['wordpress', 'jetpack'] : [get_app_name_option!(options)]
     versions = options[:version].nil? ? [android_get_release_version()] : [options[:version]]
 
+    download_signed_apks_from_google_play()
```

Comment:
To be consistent it would make sense to pass the same `app:` option add the lane to this call too, so that if `create_gh_release` is called with a specific `app:` value (even if I'm guessing this is rarely—if ever—the case in practice) then we also _only_ download just the corresponding apk, not both apks.

```suggestion
    download_signed_apks_from_google_play(app: options[:app])
```

Reply:
Thanks for catching that. Applied the suggestion.